docs/src/api_bindings.md

```diff
@@ -23,553 +23,553 @@ documented here.
 ---
 
 ## [`H5`](https://portal.hdfgroup.org/display/HDF5/Library) — General Library Functions
-- [`h5_close`](@ref HDF5.h5_close)
-- [`h5_dont_atexit`](@ref HDF5.h5_dont_atexit)
-- [`h5_free_memory`](@ref HDF5.h5_free_memory)
-- [`h5_garbage_collect`](@ref HDF5.h5_garbage_collect)
-- [`h5_get_libversion`](@ref HDF5.h5_get_libversion)
-- [`h5_is_library_threadsafe`](@ref HDF5.h5_is_library_threadsafe)
-- [`h5_open`](@ref HDF5.h5_open)
-- [`h5_set_free_list_limits`](@ref HDF5.h5_set_free_list_limits)
+- [`API.h5_close`](@ref API.h5_close)
```

Comment:
You should be able to just set `CurrentModule = HDF5.API`

Reply:
Apparently, you had to also had to remove the exiting `HDF5` prefix to get  `CurrentModule = HDF5.API` to work.